client/angular/services/columnAssignmentService.js

```diff
@@ -6,13 +6,47 @@
     columnAssignmentService.$inject = ["tweetInfoService"];
 
     function columnAssignmentService(tweetInfoService) {
+
+        var store = {};
+
         return {
             ColumnData: ColumnData,
+            assignDisplayColumns: assignDisplayColumns,
+            clearStore: clearStore,
             assignColumns: assignColumns,
```

Comment:
assignDisplayColumns and clearStore are used in the controller but yeah it's fine, it just confused me to have both assignColumns and assignDisplay columns and now the word assign has lost all meaning to me docker docker docker